package.json

```diff
@@ -81,7 +81,7 @@
     "webpack-merge": "^4.1.3"
   },
   "dependencies": {
-    "interactjs": "1.6.2",
+    "interactjs": "^1.6.2",
```

Comment:
Keep in mind this could pull the latest interactjs version - as they dont bump major versions very often.
Would be good to check the package still works with the latest version (v1.8.4)

Reply:
Have updated the packages and run the script, seems to be working find from my naive eyes.
from `yarn.lock`
```4710   │ interactjs@^1.6.2:
4711   │   version "1.8.4"```